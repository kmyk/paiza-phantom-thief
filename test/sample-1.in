3
90 100
40 15
30 30
