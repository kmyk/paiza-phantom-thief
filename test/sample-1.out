40 15
30 30
90 100
